DDRec/src/CellIDPositionConverter.cpp

```diff
@@ -83,6 +83,7 @@ namespace DD4hep {
 
 
 
+
```

Comment:
It is ok to use it as long as your branch is not merged. Those two commits shoud be squashed.
And you can fix the typo in the next commit message